nautobot/core/tests/runner.py

```diff
@@ -72,7 +92,7 @@ def teardown_databases(self, old_config, **kwargs):
         if settings.TEST_USE_FACTORIES and old_config:
             for connection in old_config:
                 db_name = connection[0].alias
-                print(f'Emptying test database "{db_name}"...')
+                print(f'Flushing test database "{db_name}"...')
                 call_command("flush", "--no-input", "--database", db_name)
```

Comment:
Does this also need a retry loop added?

Reply:
Teardown is done in different times, I didn't experience issues with that.